docs/user-guide/zos-components-installation-checklist.md

```diff
@@ -1,33 +1,64 @@
 
 ## Installing the Zowe z/OS runtime
 
-Use one of the following installation options to install Zowe z/OS components.
+Choose from the following installation options to install Zowe server-side components for z/OS.
 
 | Task | Results | Time Estimate |  
 |--------------------|----|------|
-| **Option 1: [Install Zowe runtime from a convenience build](../user-guide/install-zowe-zos-convenience-build.md)**  <br />  **Option 2: [Install Zowe with SMP/E](../user-guide/install-zowe-smpe.md)** <br /> **Option 3: [Install Zowe with a portable software instance](../user-guide/install-zowe-pswi.md)** <br />                      | Executables and binaries are unpaxed on mainframe                |   1 hour                     |
+|  **Option 1: [Install Zowe with SMP/E](./install-zowe-smpe)** <br /><br /> **Option 2: [Install Zowe with z/OSMF from a portable software instance](./install-zowe-pswi)** <br /><br /> **Option 3: [Install Zowe SMP/E build with z/OSMF workflow](./install-zowe-smpe-zosmf-workflow)** <br /><br /> **Option 4: [Install Zowe via a convenience build (PAX file)](../user-guide/install-zowe-zos-convenience-build.md)**                     | Executables and binaries are unpaxed on the mainframe                |   1 hour                     |
 
-## Configuring Zowe z/OS Components
+## Configuring Zowe z/OS Components 
+
+Choose the following options to initialize  Zowe z/OS runtime:
 
 | Task | Results | Time Estimate |  
 |-----------------------|----|------|
-| 1. Use one of the following options to initialize Zowe z/OS runtime.<br />**Option 1: [Initialize Zowe manually using zwe init command group](../user-guide/initialize-zos-system.md)  <br />** **Option 2: [Configure Zowe with z/OSMF workflows](../user-guide/configure-zowe-zosmf-workflow.md)** <br /> <br /> 2. [Configure the z/OS system for Zowe](../user-guide/configure-zos-system.md)<br />3. [Grant users permission to access z/OSMF](../user-guide/grant-user-permission-zosmf.md) <br />4. [Install and configure the Zowe cross memory server (ZWESISTC)](../user-guide/configure-xmem-server.md) | * All datasets are created and populated. See [Initialize MVS Datasets Results](../user-guide/initialize-mvs-datasets.md#results)<br /> * Started tasks are copied to system libraries<br /> * APF authorization is granted to load libraries<br /> * Zowe user is created and is asigned all required permissions<br /> * Your z/OS and security product are configured| 4 hours| 
+|  **Option 1: [Configure Zowe with zwe init](../user-guide/initialize-zos-system.md)**  <br /><br /> **Option 2: [Configure Zowe with z/OSMF workflows](../user-guide/configure-zowe-zosmf-workflow.md)**  | * All datasets are created and populated. <br /> * Started tasks are copied to system libraries  <br /><br />**Important:** Security administrator permissions are required for some zwe init sub-commands to pass.<br /> <br /> See [Initialize MVS Datasets Results](../user-guide/initialize-mvs-datasets.md#results). | 1 hour
```

Comment:
`See [Initialize MVS Datasets Results](../user-guide/initialize-mvs-datasets.md#results)`
This part is not relevant as MVS datasets are just one part of configuring, and this link is referring to article that is not even visible in the sidebar